app/views/users/show.html.erb

```diff
@@ -9,7 +9,7 @@
     <div class="d-flex justify-content-end" style="width: 350px">
       <p class="card-text">
         <small class="text-muted"
-          >Number of posts: <%= @user.postsCounter %></small
+          >Number of posts: <%= @user.posts_counter %></small
```

Comment:
It seems this change has introduced a bug to your application. When I run the application, it throws an error when I try navigating to the user show page. The reason is that you have mistyped it, it should not be `@user.posts_counter` but it should be `@user.postsCounter`.

- [ ] It will be great to revert the change made here as it is causing a bug to the application